src/app/components/header/header.component.ts

```diff
@@ -34,7 +34,8 @@ export class HeaderComponent {
   mapPathToTitle(path: string): string {
     const titleMap: { [key: string]: string } = {
       points: 'My Points',
+      repository: 'Repository',
     };
-    return titleMap[path] || 'My Workouts';
+    return titleMap[path] || 'My Calendar';
```

Comment:
Why did you change it to my calendar?

Reply:
probably something happened when  i merged the main